pkg/queryfrontend/roundtrip.go

```diff
@@ -40,8 +40,13 @@ func NewTripperWare(
 	queriesCount.WithLabelValues(labelQuery)
 	queriesCount.WithLabelValues(labelQueryRange)
 
+	overrides, err := validation.NewOverrides(*config.CortexLimits, nil)
```

Comment:
Umm, is it better to rename `overrides` to just `limits`? `overrides` is only meaningful in Cortex.

Reply:
updated